dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1429,11 +1444,15 @@ class PerfTest {
       // TODO(liyuqian): Remove isAndroid restriction once
       // https://github.com/flutter/flutter/issues/61567 is fixed.
       final bool isAndroid = deviceOperatingSystem == DeviceOperatingSystem.android;
+      if (saveTraceFile) {
+        final String logFileName = path.join(testDirectory, 'build', '$traceFilename.json');
+        copyToOutputLogs(logFileName);
+      }
       return TaskResult.success(
         data,
         detailFiles: <String>[
           if (saveTraceFile)
-            '${_testOutputDirectory(testDirectory)}/$traceFilename.json',
+            logFileName,
```

Comment:
This `logFileName` now refers to the `$resultFilename.json` from line 1415 instead of `$traceFilename.json`. We probably need to move line 1448 outside of the `if` to fix this?

Reply:
Maybe give the variable names more specific names (e.g. resultFileName, traceFileName, etc.) to avoid this somewhat hard to spot bug?